templates/artist/artist_list.html

```diff
@@ -95,7 +95,10 @@
     <div class="row small-up-1 medium-up-2 large-up-4 artistlist">
         {% for artist in artists %}
             <div class="column">
-                {% include "artist/includes/artist_preview.html" with artist=artist %}
```

Comment:
Yeah, I created a new brand from a previous branch which I'm now realizing was not the right way to do it